processor/sourcemap/decoder.go

```diff
@@ -19,14 +20,14 @@ func DecodeSourcemapFormData(req *http.Request) (map[string]interface{}, error)
 	}
 	defer file.Close()
 
-	var parsedSourcemap map[string]interface{}
-	err = json.NewDecoder(file).Decode(&parsedSourcemap)
+	var sourcemap bytes.Buffer
+	_, err = io.Copy(&sourcemap, file)
```

Comment:
alternatively, I think you can do:
`buf, err := ioutil.ReadAll(file)` and use `buf` below.